Add Rating component (#5938)
jpmorganchase/salt-ds: site/docs/components/rating/usage.mdx

---
title:
  $ref: ./#/title
layout: DetailComponent
sidebar:
  exclude: true
data:
  $ref: ./#/data
---

## Using the components

### When to use

- When you have to provide feedback or indicate your level of satisfaction with a product, service, or experience.

### When not to use

> - When you have fewer than 3 increments (such as a binary thumbs up/down or like/dislike) do not provide enough nuance for most rating scenarios. In this scenario, use a [`ToggleButtonGroup`](../toggle-button).

Reviewer: ```suggestion
- When you have fewer than 3 increments (such as a binary thumbs up/down or like/dislike) don't provide enough nuance for most rating scenarios. In this scenario, use a [`ToggleButtonGroup`](../toggle-button).
```
